python_ta/config/__init__.py

```diff
@@ -82,18 +82,26 @@ def load_messages_config(path: str, default_path: str, use_pyta_error_messages:
             logging.warning(f"Could not find messages config file at {str(Path(path).resolve())}.")
             merge_from = {}
 
+    # The TOML file has section headers, parse it to get rid of section headers
+    if merge_from != {} and "pylint" in list(merge_from.keys())[0]:
```

Comment:
This check isn't totally correct, as it's possible for the user to only be customizing PythonTA custom checker messages, not pylint's messages.

Looking at this some more, I think the easiest approach is to _always_ ignore all section headers, regardless of what they are. You could write a `flatten` function which takes a nested dictionary and turns it into a flat dictionary. This will also simplify the documentation: we can remove all of the old stuff and just say that all section headers will be ignored. (This still preserves backwards compatibility!)